src/zabapgit.prog.abap

```diff
@@ -48,7 +48,6 @@ INCLUDE zabapgit_authorizations.
 INCLUDE zabapgit_exit.
 INCLUDE zabapgit_proxy.
 
-INCLUDE zabapgit_tag.
```

Comment:
keep the INCLUDE, like eg. https://github.com/larshp/abapGit/blob/master/src/zabapgit_dot_abapgit.prog.abap